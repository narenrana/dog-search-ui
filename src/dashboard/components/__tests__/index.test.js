/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import Dashboard from "../index";

import { Provider } from "react-redux";
import configureStore from "../../../configure-store";

const store = configureStore();

describe("Dashboard test", () => {
  it("Dashboard should match snapshot", () => {
    const component = renderer.create(
      <Provider store={store}>
        <Dashboard />
      </Provider>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
