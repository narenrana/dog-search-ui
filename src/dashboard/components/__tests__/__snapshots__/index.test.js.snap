// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard test Dashboard should match snapshot 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiBox-root MuiBox-root-2"
  >
    <div
      className="MuiGrid-root MuiGrid-container"
    />
  </div>
</div>
`;
